### DISCUSSION TITLE
Need help with compiling errors

### DISCUSSION THREAD

**Discussion by EnterJokeHere1** [FunkinCrew/Funkin#1967] [Compiling help]
I try to compile but it gives me these errors:

Warning: Could not find Visual Studio 2017 VsDevCmd
Missing HXCPP_VARS
Error: Could not automatically setup MSVC

**Comment by gdandsm64fan123** [comment]
bro u need to use 2019 u stupid or smth?

**Reply by gdandsm64fan123** [reply]
but strangely enough i get that error too
